src/frontend/src/views/analysis/Analysis.vue

```diff
@@ -8,7 +8,7 @@
       style="padding: 12px; margin: 12px"
     >
       <v-card-title>감정 그래프</v-card-title>
-      <radar-chart />
+      <radar-chart v-if="loaded" :chartdata="chartdata" :options="options" />
```

Comment:
```suggestion
      <radar-chart />
```

`Analysis.vue`에 레이더 차트에 대한 로직과 변수들(`loaded`, `chartdata`, `options`)이 있는데 그걸 `RadarChart.vue`로 옮겨서 위처럼 표현되는데 좋을듯해요!!

Reply:
그래가 제안한대로 시도해보는데 아직 뷰 다루는 게 미숙해서 그런지 잘 안되네요 😭 
우선 [vue-chartjs 공식 문서](https://vue-chartjs.org/guide/#chart-with-api-data)대로 구현한 다음에 뷰엑스를 적용해본거라...
혹시 간단히 적용되는 거라면 알려주시면 감사하겠습니다 🙇🏻‍♀️

Reply:
엇 한번 해볼게요!!

Reply:
비동기 문제도 겹치고 그래서 어렵네요;;ㅎㅎㅎ
다음에 실력이 더 나아지면 다시 시도해봐야겠네요 ㅠㅠ